### PR TITLE
added start a pipeline (#9)

### PR DESCRIPTION
**Pull request by lstocchi**
it is related to #9 

This is a basic implementation to start a pipeline. It just start it without allowing user to set any input value. So the pipeline has to be complete without any input needed. As soon as the form to run a task is accepted i'll add it here as well.

### CODE REVIEW

File: src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.redhat.devtools.intellij.tektoncd.actions.component;
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java]: Missing header

File: src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    @Override
+    public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object selected, Tkn tkncli) {
+        String namespace = ((PipelineNode)selected).getParent().getParent().toString();
+        try {
+            tkncli.startPipeline(namespace, selected.toString());
+            ((PipelineNode)selected).reload();
+        } catch (IOException e) {
+            // give a visual notification to user if an error occurs
+            Notification notification = new Notification("SaveNotification", "Error", e.getLocalizedMessage(), NotificationType.ERROR);
+            notification.notify(ProjectManager.getInstance().getDefaultProject());
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java]: Not sure the defaultProject is usefull for that. Use `Notifications.Bus.notify` to display your notification.

File: src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Override
+    public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object selected, Tkn tkncli) {
+        String namespace = ((PipelineNode)selected).getParent().getParent().toString();
+        try {
+            tkncli.startPipeline(namespace, selected.toString());
+            ((PipelineNode)selected).reload();
+        } catch (IOException e) {
+            // give a visual notification to user if an error occurs
+            Notification notification = new Notification("SaveNotification", "Error", e.getLocalizedMessage(), NotificationType.ERROR);
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/actions/component/StartPipelineAction.java]: The first argument identifies the plugin. So should be 'Tekton Pipelines' and should be a constant

**Comment by lstocchi** [comment]
@jeffmaury PR rebased. Whenever you can you can give it a look. Thanks

**Pull request merged**